src/components/AlertWarning/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ComponentType } from 'react';
+
+interface AlertWarningProps {
+  Icon: ComponentType;
+  errorMessage: string[];
+}
+
+const AlertWarning = ({ Icon, errorMessage }: AlertWarningProps): JSX.Element => {
+  return (
+    <div className="alert">
+      <div className="alert-content">
+        <div className="alert__icon">
+          <Icon />
+        </div>
+        <div className="alert-header-title">Error</div>
```

Comment:
I'm not really seeing the use for the `alert-content` class, perhaps we can simplify the HTML so that it follows BEM better 👍 

```suggestion
    <div className="alert">
      <div className="alert__icon">
        <Icon />
      </div>
      <div className="alert__title">Error</div>
```

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/8172c6c9c8c48e41c595245d1a91cef8b76650c7